code/modules/projectiles/projectile/ego_bullets/waw.dm

```diff
@@ -1,4 +1,4 @@
 /obj/projectile/ego_bullet/ego_correctional
 	name = "correctional"
-	damage = 6 // Firing in a spread of 6
+	damage = 15 // supposed to fire in a spread of 6 at 6 damage. This will do for now
```

Comment:
This looks like a giant nerf, even if temporary...

Reply:
Shotguns literally don't work, egor.
If they did I would not nerf it.

It's better to have a giant nerfed ego than a broken and bad one